Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM registry.access.redhat.com/ubi8/go-toolset:1.19.10-10 as builder
+FROM registry.access.redhat.com/ubi8/go-toolset:1.19.10-16 as builder
```

Comment:
I think this change helps, going from the 2-month old 1.19.10-10 to the newer one, but the biggest vulnerability boost comes from the second FROM line where changing to ubi-minimal:8.8 fixes all the OS items.